go/couler/submitter/argo_submitter.go

```diff
@@ -16,6 +17,14 @@ type ArgoWorkflowSubmitter struct {
 	kubeConfigPath string
 }
 
+// New returns ArgoWorkflowSubmitter struct
+func New(namespace, kubeConfigPath string) *ArgoWorkflowSubmitter {
+	return &ArgoWorkflowSubmitter{
```

Comment:
What's the use case for this?

Reply:
The exposed `New` function allows other packages to use `ArgoWorkflowSubmitter`, c.f. the Couler service in the internal repo.